src/query/query_aggregate.cpp

```diff
@@ -999,11 +999,11 @@ qdata_evaluate_aggregate_optimize (cubthread::entry *thread_p, cubxasl::aggregat
     case PT_COUNT:
       if (agg_p->option == Q_ALL)
 	{
-	  db_make_int (agg_p->accumulator.value, oid_count - null_count);
+	  db_make_bigint (agg_p->accumulator.value, oid_count - null_count);
 	}
       else
 	{
-	  db_make_int (agg_p->accumulator.value, key_count);
+	  db_make_bigint (agg_p->accumulator.value, key_count);
```

Comment:
Oid_count, null_count, key_count are integer.
I think the aggregate_optimize function seems to be deprecated after version 10, so it is okay to keep this.